app/src/androidTest/java/com/example/crosswordToLearn/ChooseTopicsInstrumentedTest.kt

```diff
@@ -22,18 +18,22 @@ class ChooseTopicsInstrumentedTest {
     @get:Rule
     var activityTestRule: ActivityScenarioRule<MainActivity> =
         ActivityScenarioRule(MainActivity::class.java)
-
+    
     @Rule
     @JvmField
     val retryTestRule = RetryTestRule()
 
+    @Rule
+    @JvmField
+    var timeout: Timeout = Timeout.millis(30000)
```

Comment:
Please use constant inside Utils for this timeout. And also add comment near it why we need to use this constant

Reply:
https://medium.com/swlh/kotlin-basics-of-companion-objects-a8422c96779b Something like this also can see example inside project.

Reply:
hm, I use it not only inside class, i.e. https://github.com/omen273/crosswordApp/pull/103#discussion_r599452108
How should I use companion object in this case?

Reply:
Two variants: create common base class for all test classes or use special class for test constants.

Reply:
Please see MainActivity.kt for example